util/build_configs/cray-internal/generate-modulefile.bash

```diff
@@ -72,10 +72,43 @@ conflicts xt-mpich2 < 6.0.0
 conflict chapel
 #conflict gcc
 
+##
+## Now, we're going to attempt to get to the right build type
+###
+#
+set network seastar
```

Comment:
Seems a little odd that we're still defaulting to seastar

Reply:
We can clean this up later; for now I'm trying to make the smallest change(s) that lead to success.